createUserFeatureFlag/go.mod

```diff
@@ -0,0 +1,21 @@
+module createUserFeatureFlag
+
+go 1.20
+
+require (
+	github.com/Real-Dev-Squad/feature-flag-backend v0.0.0-00010101000000-000000000000
```

Comment:
Hi @shubham-y this version is coming because of the replace directive, can you correct using this [REF](https://stackoverflow.com/questions/58012771/go-mod-fails-to-find-version-v0-0-0-00010101000000-000000000000-of-a-dependency#:~:text=0.0%2D00010101000000%2D000000000000%20is%20what,you%20using%20that%20long%20version.)

Reply:
Thanks for the reference. Updated the code